docs/production/mobile-push-notifications.md

```diff
@@ -176,3 +176,60 @@ push notifications through the new app is quite straightforward:
 * Restart the Zulip server.
 
 [apple-docs]: https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html
+
+
+# Removing sent notifications from mobile as messages read on server
```

Comment:
These docs probably belong somewhere in `docs/subsystems/`; this page is user-facing docs.